docs/user/rest-api.rst

```diff
@@ -0,0 +1,282 @@
+Rest API Reference
+==================
+
+.. _monitoring_live_documentation:
+
+Live documentation
+------------------
+
+.. image:: https://raw.githubusercontent.com/openwisp/openwisp-monitoring/docs/docs/api-doc.png
+    :target: https://raw.githubusercontent.com/openwisp/openwisp-monitoring/docs/docs/api-doc.png
+
+A general live API documentation (following the OpenAPI specification) at
+``/api/v1/docs/``.
+
+.. _monitoring_browsable_web_interface:
+
+Browsable web interface
+-----------------------
+
+.. image:: https://raw.githubusercontent.com/openwisp/openwisp-monitoring/docs/docs/api-ui-1.png
+    :target: https://raw.githubusercontent.com/openwisp/openwisp-monitoring/docs/docs/api-ui-1.png
+
+.. image:: https://raw.githubusercontent.com/openwisp/openwisp-monitoring/docs/docs/api-ui-2.png
+    :target: https://raw.githubusercontent.com/openwisp/openwisp-monitoring/docs/docs/api-ui-2.png
+
+Additionally, opening any of the endpoints :ref:`listed below
+<monitoring_rest_endpoints>` directly in the browser will show the
+`browsable API interface of Django-REST-Framework
+<https://www.django-rest-framework.org/topics/browsable-api/>`_, which
+makes it even easier to find out the details of each endpoint.
+
+.. _monitoring_rest_endpoints:
+
+List of endpoints
+-----------------
+
+Since the detailed explanation is contained in the :ref:`Live
+documentation <monitoring_live_documentation>` and in the :ref:`Browsable
```

Comment:
in cases like here, where the target heading has the same title of the link text, we can just shorten this, eg: `:ref:`<monitoring_live_documentation>`. This avoids some redundancy.